controllers/Index.php

```diff
@@ -190,6 +190,46 @@ public function onCreateObject()
         return $result;
     }
 
+    public function onDuplicateObject()
+    {
+        $this->validateRequestTheme();
+
+        $type = Request::input('objectType');
+
+        $object = $this->loadObject($type, trim(Request::input('objectPath')));
+        $parent = Request::input('parent');
+        $parentPage = null;
+
+        if ($type == 'page') {
```

Comment:
@der-On Please use `===` for string comparisons.

Reply:
This was copy pasted from `onCreateObject` ;) but sure I will.